website_cookie_notice/models/website.py

```diff
@@ -0,0 +1,11 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Website(models.Model):
+    _inherit = "website"
+
+    cookie_notice_legal_page_id = fields.Many2one(
+        comodel_name="website.page", string="Linked legal page"
```

Comment:
This is wrong IMHO.

If you use a page here, there's no way of adding an `#anchor` to it. As discussed in #670, this is most likely the path to follow for the future.

Also I don't think we really need to go for res.settings to change the link. We already have the website editor which lets users do that.

So I think the best is to remove all models and just provide a basic template.